src/toil/common.py

```diff
@@ -617,101 +839,111 @@ def addOptions(parser: ArgumentParser, config: Optional[Config] = None, jobstore
                     "as well as parameters to control the level of provisioning."
     )
     provisioner_choices = ['aws', 'gce', None]
+
     # TODO: Better consolidate this provisioner arg and the one in provisioners/__init__.py?
     autoscaling_options.add_argument('--provisioner', '-p', dest="provisioner", choices=provisioner_choices,
+                                     default=None,
                                      help=f"The provisioner for cluster auto-scaling.  This is the main Toil "
                                           f"'--provisioner' option, and defaults to None for running on single "
                                           f"machine and non-auto-scaling batch systems.  The currently supported "
-                                          f"choices are {provisioner_choices}.  The default is {config.provisioner}.")
-    autoscaling_options.add_argument('--nodeTypes', default=None,
+                                          f"choices are {provisioner_choices}.  The default is %(default)s.")
+    autoscaling_options.add_argument('--node_types', default=[], dest="node_types", type=parse_one_node_type, nargs="*",
```

Comment:
It looks like this is meant to now be used like `--node_types t2.large c5.4xlarge`. But the documentation now shows you passing a bracket-enclosed list that I don't think `parse_one_node_type` can actually parse. Also, using `nargs="*"` can cause trouble because it doesn't really know when to stop consuming arguments. If I have `toil-cwl-runner --logDebug script.cwl inputs.json` and I want to set a node type and I do `toil-cwl-runner --logDebug --node_types c5.4xlarge script.cwl inputs.json`, then suddenly my positional arguments have been eaten.

We might want to consider `action="append"` here instead, and a singular option that can repeat. Like `--node_type t2.large:1.5 --node_type c5.4xlarge/c5a.4xlarge:0.42`.

Either way we have to make sure to fix the docs/examples to show it the way that actually works on the command line, and not with square brackets.